### PR TITLE
RA-1 Project setup

### PR DESCRIPTION
**Pull request by ShilpiMaurya**
#  Ticket
https://trello.com/c/H8s6a8gm/4-ra-1-project-setup
# Changes done
- Added resume.json

### CODE REVIEW

File: resume.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+          "GitFlow",
+          "SEO",
+          "React"
+        ]
+      }
+    ],
+    "languages": [
+      {
+        "language": "English",
+        "fluency": "Native speaker"
```
Comment (vaibhavKumarTokyo) [inline resume.json]: Suggestion:
Native speaker -> Native

File: resume.json [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      },
+      {
+        "language": "Japanese",
+        "fluency": "Elementary Proficiency(N5)"
```
Comment (vaibhavKumarTokyo) [inline resume.json]: Suggestion:
`Elementary Proficiency(N5)` -> `Elementary (N5)`

File: resume.json (hunk 1 above)
Comment (ShilpiMaurya) [reply resume.json]: fixed

File: resume.json (hunk 2 above)
Comment (ShilpiMaurya) [reply resume.json]: fixed.

**Pull request merged**